client/src/index.tsx

```diff
@@ -30,18 +30,20 @@ if (REMOTE_STORAGE_URI !== undefined) {
 
 // YOLO-routing
 const match = /^#\/login\?userId=([a-zA-Z0-9\-]+)$/.exec(window.location.hash)
+let userId: string | undefined
 if (match) {
-    const userId = match[1]
-    console.log(`Logging in with userId=${userId}`)
+    userId = match[1]
     const expiry = new Date()
     expiry.setTime(expiry.getTime() + (365 * 24 * 60 * 60 * 1000))
     document.cookie = `userId=${userId}; expires=${expiry.toUTCString()}`
     window.location.href = "/"
+} else if (document.cookie) {
+    userId = document.cookie.split("=").pop()
```

Comment:
This will break if userId is not the first cookie. Might be easier to use something like https://github.com/js-cookie/js-cookie

Reply:
Also this will break if there's no cookie?

Reply:
Cool, yes.  Just implemented it.